Unskip WPT tests in upgrade-insecure-requests and mixed-content folders

pvollan opened the pull request:
#### 23785dbffa08a6d45bfda3233f2f66773fd02fc9
<pre>
Unskip WPT tests in upgrade-insecure-requests and mixed-content folders
<a href="https://bugs.webkit.org/show_bug.cgi?id=301034">https://bugs.webkit.org/show_bug.cgi?id=301034</a>
<a href="https://rdar.apple.com/162931588">rdar://162931588</a>

Reviewed by Sihui Liu.

We can unskip these tests after having enabled the local DNS resolver.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/sharedworker-classic.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/sharedworker-module.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/link-prefetch-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/object-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/picture-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/worker-import-data.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/worker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.meta/opt-in/link-prefetch-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.meta/opt-in/object-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.meta/opt-in/picture-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.meta/unset/link-prefetch-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.meta/unset/object-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/worker-classic-data.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/worker-classic.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/worker-module-data.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/worker-module.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/unset/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/unset/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/unset/worker-import-data.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/unset/worker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/unset/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/upgrade/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/upgrade/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/unset/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/unset/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/unset/worker-import-data.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/unset/worker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/unset/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/upgrade/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/upgrade/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.http-rp/upgrade/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.http-rp/upgrade/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/unset/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/unset/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/unset/worker-import-data.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/unset/worker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/upgrade/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/upgrade/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/worker-classic-data.meta/unset/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/worker-module-data.meta/unset/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/link-upgrade.sub.https-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/sharedworker-import.https-expected.txt: Added.

Canonical link: <a href="https://commits.webkit.org/301818@main">https://commits.webkit.org/301818@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/8eb6c2b79fec5ceddbd0c071f04d38f1365e8661

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/127148 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/46783 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/37919 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/134212 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/78703 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/ebdc0ae3-9a53-4a75-aa88-9c2dda59fe50) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/47398 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/55309 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/96758 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/64801 "Passed tests") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/3ae15a0b-9d0a-4592-917f-d82acc85a94b) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/130096 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/37944 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/113879 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/77264 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/f2c6d2ed-bd8b-4d70-a499-64172ae81bd0/49a05baf-a762-4428-b0e2-092579e723fd) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/36816 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/31996 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/77592 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/107816 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/32360 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/136695 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/53800 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/41448 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/105278 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/54308 "Built successfully") | [❌ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/110231 "layout-tests (failure) (timed out)") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/104964 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/50494 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/28934 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/51363 "Built successfully") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/19891 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/53731 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/52964 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/56300 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/54727 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

szewai reviewed (approved):
r=me if bots are happy

pvollan committed:
Unskip WPT tests in upgrade-insecure-requests and mixed-content folders
https://bugs.webkit.org/show_bug.cgi?id=301034
rdar://162931588

Reviewed by Sihui Liu.

We can unskip these tests after having enabled the local DNS resolver.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/sharedworker-classic.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/sharedworker-module.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/link-prefetch-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/object-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/picture-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/worker-import-data.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/worker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.meta/opt-in/link-prefetch-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.meta/opt-in/object-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.meta/opt-in/picture-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.meta/unset/link-prefetch-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/top.meta/unset/object-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/worker-classic-data.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/worker-classic.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/worker-module-data.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/mixed-content/gen/worker-module.http-rp/opt-in/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/unset/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/unset/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/unset/worker-import-data.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/unset/worker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/unset/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/upgrade/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/iframe-blank-inherit.meta/upgrade/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/unset/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/unset/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/unset/worker-import-data.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/unset/worker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/unset/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/upgrade/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/srcdoc-inherit.meta/upgrade/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.http-rp/upgrade/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.http-rp/upgrade/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/unset/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/unset/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/unset/worker-import-data.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/unset/worker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/upgrade/iframe-tag.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/top.meta/upgrade/sharedworker-import.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/worker-classic-data.meta/unset/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/gen/worker-module-data.meta/unset/xhr.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/upgrade-insecure-requests/link-upgrade.sub.https-expected.txt: Added.
* LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/mixed-content/gen/top.http-rp/opt-in/sharedworker-import.https-expected.txt: Added.

Canonical link: https://commits.webkit.org/301818@main
